run/test/specs/user_actions.spec.ts

```diff
@@ -357,7 +359,9 @@ async function setNicknameIos(platform: SupportedPlatformsType) {
   await device1.clickOnElement("More options");
   // Click on username to set nickname
   await device1.clickOnElement("Username");
-  await device1.deleteTextIos("Username");
+  await sleepFor(1000);
+  await device1.clickOnElement("Username");
+  await device1.deleteText("Username");
```

Comment:
Because we need to get the little dialog up that has 'Select All' and to do that I need to essentially double click but its like a click wait click. It's the only combination of actions that i've found that allow text deletion...